src/pages/profile/index.js

```diff
@@ -0,0 +1,66 @@
+import { useEffect, useRef } from "react";
+import { useHistory } from "react-router";
+import { SubmitButton } from "../../components/elements";
+import { AUTH } from "../../constants";
+import { getUserData, updateUserData } from "../../services/userService";
+import "./profile.css";
+
+export function Profile() {
+  const history = useHistory();
+  const nameRef = useRef();
+  const emailRef = useRef();
+  const formRef = useRef();
+  useEffect(() => {
+    getUserData()
+      .then((userData) => {
+        nameRef.current.value = userData.name;
+        emailRef.current.value = userData.email;
+      })
+      .catch((errorMessage) => {
+        if (errorMessage === "You are not logged in") history.replace(AUTH);
+      });
+  });
```

Comment:
You are using `useEffect` without any dependency array, which means it will fetch the user on each re-render. Pass an empty array as a dependency array to fetch user information only on the first render. It will be great if you can consider visual indent it helps me to identify separate related code blocks quickly and it will also help you to understand your code when you will read it after some time.